README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Purpose
 
-_... help fill this in!_
+* Define and maintain security policies and procedures for:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+* Define and maintain policies and procedures for the coordination of security concerns within the external Node.js open source ecosystem
```

Comment:
It would certainly be good to have a process in place to help developers mitigate those node related security issues. This, of course could take quite a few forms from helping with PRs to writing up guides and documentation